Sources_v3/StreamChatUI/Common Views/ChatSquareButton.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatSquareButton: Button {
+    open var defaultSize: CGFloat?
+    
+    public init(image: UIImage? = nil, tintColor: UIColor? = nil) {
+        super.init(frame: .zero)
+        setImage(image, for: .normal)
+        self.tintColor = tintColor
+    }
```

Comment:
Let's try to not override initializers. It makes the view impossible to initialize via IB.